web/templates/web/study-detail.html

```diff
@@ -31,23 +30,85 @@
     <script>
         function childSelected(selected) {
             var participateButton = document.getElementById('participate-button');
-            selected.value === 'none' ? participateButton.disabled = true : participateButton.disabled = false;
+            if (selected.value === 'none') {
+                participateButton.disabled = true;
+                document.getElementById('too-old').style.display = 'none';
+                document.getElementById('too-young').style.display = 'none';
+            } else {
+                participateButton.disabled = false;
+            }
             participateButton.value = selected.value;
+            modifyTemplateWithAgeCheck(selected);
+        }
+
+        function modifyTemplateWithAgeCheck(selected) {
+            document.getElementById('too-old').style.display = 'none';
+            document.getElementById('too-young').style.display = 'none';
+            var birthday = selected.value.split('|')[1];
+            var age = calculateAgeInDays(birthday);
+            if (!isEligible(age)) {
+                if (isOldEnough(age)) {
+                    document.getElementById('too-old').style.display = 'inline-block';
+                } else {
+                    document.getElementById('too-young').style.display = 'inline-block';
+                }
+            }
         }
 
         function portToParticipate(selected) {
-            window.location.href = "/studies/" + "{{ study.uuid }}" + '/' + selected.value;
+            window.location.href = "/studies/" + "{{ study.uuid }}" + '/' + selected.value.split('|')[0];
```

Comment:
Right, I just meant you could still use the template engine like:
`window.location.href = '{% url 'study-detail' study.uuid %}' + '/' + selected.value.split('|')[0];`

There might be too many slashes in the middle, don't remember if `url` will put a slash on the end or not. Also, I just found out that this needs a trailing slash on the URL, at least in my celery branch, or it gets a 403 from Google.